package.json

```diff
@@ -73,6 +73,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
+    "esm": "^3.2.25",
```

Comment:
What for is this change?

Reply:
unfortunately mocha can not run tests with 'import' but our unit tests use it. The solution is to use this tool
https://stackoverflow.com/questions/46487307/is-it-possible-to-use-es6-modules-in-mocha-tests/60522428#60522428